sys/include/net/ng_netif/hdr.h

```diff
@@ -134,6 +136,46 @@ static inline void ng_netif_hdr_set_dst_addr(ng_netif_hdr_t *hdr, uint8_t *addr,
     memcpy(((uint8_t *)(hdr + 1)) + hdr->src_l2addr_len, addr, addr_len);
 }
 
+/**
+ * @brief   Builds a generic network interface header for sending and
```

Comment:
No that's in #2553. This just offers `netif_hdr_init()` and `pktbuf_add()` in one go.